package.json

```diff
@@ -2,12 +2,14 @@
   "name": "@jupiterone/jupiterone-client-nodejs",
   "version": "0.21.1",
   "description": "A node.js client wrapper for JupiterOne public API",
-  "main": "src/j1client.js",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "repository": "git@github.com:JupiterOne/jupiterone-client-nodejs.git",
   "author": "JupiterOne <dev@jupiterone.io>",
   "license": "MIT",
   "files": [
-    "src",
+    "dist",
+    "!dist/**.test.*",
```

Comment:
I think you want to prevent tests from being compiled in the tsconfig as opposed to excluding them here.

Reply:
Ahh yeah. That would be better. Doing that now.

Reply:
k, created I created a tsconfig file with test exclusions and pointed the tsc command to that in the prepare step. Lemme know if there's a better way.